Makefile

```diff
@@ -130,6 +137,9 @@ format-modified:          ## Run ruff to format only modified code
 init-precommit:    		  ## install te pre-commit hook into your local git repository
 	($(VENV_RUN); pre-commit install)
 
+docker-build:
```

Comment:
That fails for me on macOS M1 (haven't looked into it):

```
Dockerfile:36
--------------------
  35 |     # Install runtime OS package dependencies
  36 | >>> RUN --mount=type=cache,target=/var/cache/apt \
  37 | >>>     apt-get update && \
  38 | >>>         # Install dependencies to add additional repos
  39 | >>>         apt-get install -y --no-install-recommends \
  40 | >>>             # Runtime packages (groff-base is necessary for AWS CLI help)
  41 | >>>             ca-certificates curl gnupg git make openssl tar pixz zip unzip groff-base iputils-ping nss-passwords procps iproute2 xz-utils libatomic1
  42 |
--------------------
ERROR: failed to solve: process "/bin/sh -c apt-get update &&         apt-get install -y --no-install-recommends             ca-certificates curl gnupg git make openssl tar pixz zip unzip groff-base iputils-ping nss-passwords procps iproute2 xz-utils libatomic1" did not complete successfully: exit code: 100
make: *** [docker-build] Error 1
```

Reply:
my Docker VM was running out of space hitting this issue: https://github.com/docker/for-mac/issues/6963

`docker image prune` recovered ~80GB and fixed the build 👍